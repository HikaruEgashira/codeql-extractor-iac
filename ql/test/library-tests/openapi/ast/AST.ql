private import iac::OpenApi

query predicate oapi(Document n) { any() }

query predicate oapiInfo(Info n) { any() }

query predicate oapiServers(Servers n) { any() }

query predicate oapiPaths(Path n) { any() }
