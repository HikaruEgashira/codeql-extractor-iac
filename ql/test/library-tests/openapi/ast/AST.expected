oapi
| swagger.yml:1:1:24:31 | OpenApi Document |
| uber.json:1:1:370:1 | OpenApi Document |
oapiInfo
| swagger.yml:3:3:5:17 | OpenApi Information |
| uber.json:3:13:7:5 | OpenApi Information |
oapiServers
| swagger.yml:7:3:10:87 | OpenApi Servers |
oapiPaths
| swagger.yml:12:3:24:31 | OpenApi Path |
